.coderabbit.yaml

```
@@ -0,0 +1,77 @@
+    
+  # CSS/SCSS settings
+  css:
+    enabled: true
+
+# Chat settings
+chat:
+  auto_reply: true
+
+# Early access features
+early_access:
+  enabled: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)